torch/csrc/autograd/profiler_kineto.h

```diff
@@ -203,6 +217,7 @@ struct TORCH_API KinetoEvent {
   uint8_t activity_type_ = 0;
   c10::optional<std::vector<std::vector<int64_t>>> shapes_;
   c10::optional<std::vector<std::string>> stack_;
+  c10::optional<std::string> module_hierarchy_;
```

Comment:
Do you see any future need to parse this hierarchy that calls out for having the hierarchy broken into its different levels? Basically just wondering if this should be a vector as well.
Just wondering...

Reply:
@raziel probably yes. I can imagine that we build a tool integrated with the profiler that provides hierarchical break down of latency, but that would require some post processing anyway (such as modules marked with "SELF" are the same module that was seen a level up in hierarchy).